cloud/docker/docker.py

```diff
@@ -1774,7 +1774,7 @@ def main():
             devices         = dict(default=None, type='list'),
             memory_limit    = dict(default=0),
             memory_swap     = dict(default=0),
-            cpu_shares      = dict(default=0),
+            cpu_shares      = dict(default=0, type='int'),
```

Comment:
``type='int'`` makes no sense here. Maybe I want to use ``cpu_shares: 1k`` here.